src/Calculator/Views/UnitConverter.xaml

```diff
@@ -428,14 +428,21 @@
                 </VisualState>
             </VisualStateGroup>
             <VisualStateGroup x:Name="CurrencySecondaryStatusStates">
+                <VisualState x:Name="NormalCurrencyStatus">
+                    <VisualState.Setters>
+                        <Setter Target="CurrencySecondaryStatusTextBlock.Visibility" Value="Collapsed"/>
```

Comment:
For `Normal` states, we typically set the appropriate properties directly on the element and define an empty `<VisualState x:Name="NormalCurrencyStatus"/>`

Reply:
Makes sense.